src/ui/story.js

```diff
@@ -17,6 +17,7 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     fluid.defaults("sjrk.storyTelling.story", {
         gradeNames: ["fluid.modelComponent"],
         model: {
+            id: null, // to be filled in on initialization
```

Comment:
It would be useful to document the db structure and fields including descriptions.